Formula/weboob.rb

```diff
@@ -3,10 +3,9 @@ class Weboob < Formula
 
   desc "Web Outside of Browsers"
   homepage "https://weboob.org/"
-  url "https://git.weboob.org/weboob/weboob/uploads/e8c77b143e84c97fca354579575d18e9/weboob-1.3.tar.gz"
-  mirror "https://deb.debian.org/debian/pool/main/w/weboob/weboob_1.3.orig.tar.gz"
-  sha256 "c991785c889877c76f18d19e372ed4ae0c3f8b819fd1e8da296bd34b1381be54"
-  revision 2
+  url "https://git.weboob.org/weboob/weboob/uploads/08dad6b2a68fd618ba8c87c627393d06/weboob-1.4.tar.gz"
+  mirror "https://files.pythonhosted.org/packages/34/e7/230d679030e6970be3c98339811264999c5484ad657feb9a95e3f9ce5035/weboob-1.4+1.tar.gz"
```

Comment:
Could you keep the mirror pointing to `deb.debian.org`?

Reply:
Version 1.4 isn't available yet in the Debian mirrors, and it is usually very long to get an update pushed. I have an update pending for Gentoo too, if it gets merged I could switch to their mirrors if you prefer.

It's the first time we release a tarball properly (the same pushed on pypi and provided to other users) hence the change to pypi. The other advantage is that for new versions we can update it immediately.